docs/broker/README.md

```diff
@@ -51,6 +51,28 @@ spec:
 
 ## Usage
 
+### ClusterChannelProvisioner
+
+`Broker`'s use their `spec.channelTemplate` to create their internal `Channel`s, which dictate the durability guarantees of events sent to that `Broker`. If `spec.channemTemplate` is not specified, then the [default provisioner](https://github.com/knative/docs/blob/master/docs/eventing/channels/default-channels.md#setting-the-default-channel-configuration) for their namespace is used.
```

Comment:
Change the link to https://www.knative.dev/docs/eventing/channels/default-channels/ ?

Reply:
Done.